### PR TITLE
net-p2p/go-ethereum: New package

### PR DESCRIPTION
**Pull request by MathyV**
go-ethereum is a Go client for the Ethereum cryptocurrency.

This ebuild was based on work found in the hnaparst overlay.

Package-Manager: portage-2.3.0

### CODE REVIEW

**Comment by MathyV** [comment]
@gentoo/proxy-maint

File: net-p2p/go-ethereum/go-ethereum-1.4.16.ebuild [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+src_compile() {
+	emake geth
+	use evm && emake evm
+}
+
+src_install() {
+	for d in README* COPYING* AUTHORS; do
+		[[ -s "${d}" ]] && dodoc "${d}"
+	done
```
Comment (SoapGentoo) [inline net-p2p/go-ethereum/go-ethereum-1.4.16.ebuild]: get rid of this whole for loop, and in its place just put `einstalldocs` (screw the `COPYING*` stuff)

File: net-p2p/go-ethereum/go-ethereum-1.4.16.ebuild [hunk 2]
```diff
@@ -0,0 +1,42 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+
+inherit eutils
```
Comment (SoapGentoo) [inline net-p2p/go-ethereum/go-ethereum-1.4.16.ebuild]: why do you need `eutils`?

File: net-p2p/go-ethereum/go-ethereum-1.4.16.ebuild [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+IUSE="evm"
+
+DEPEND="
+	dev-lang/go
+"
+RDEPEND="${DEPEND}"
+
+src_prepare() {
+	sed -i \
+		-e 's|\$(GOROOT)/bin/go|go|g' \
```
Comment (SoapGentoo) [inline net-p2p/go-ethereum/go-ethereum-1.4.16.ebuild]: if its small, rather use a patch

File: net-p2p/go-ethereum/go-ethereum-1.4.16.ebuild (hunk 2 above)
Comment (MathyV) [inline net-p2p/go-ethereum/go-ethereum-1.4.16.ebuild]: Apparently I don't :-) Remnant from old ebuild I guess

**Comment by MathyV** [comment]
Fixed

**Pull request closed without merging**